Userland/Libraries/LibWeb/CSS/Identifiers.json

```diff
@@ -171,4 +173,4 @@
   "xxx-large",
   "zoom-in",
   "zoom-out"
-]
\ No newline at end of file
+]
```

Comment:
Please make sure all commit messages have a `Category:` prefix.

Also, please configure your editor to add a newline at the end of files.

Reply:
Somehow clang-format removed the trailing newline in my setup. It's fixed now.